x-pack/plugins/security_solution/public/common/containers/source/use_data_view.tsx

```diff
@@ -84,15 +116,11 @@ export const useDataView = (): { indexFieldsSearch: (selectedDataViewId: string)
                     runtimeMappings: response.runtimeMappings,
                   })
                 );
-                if (searchSubscription$.current[selectedDataViewId]) {
-                  searchSubscription$.current[selectedDataViewId].unsubscribe();
-                }
+                searchSubscription$.current[selectedDataViewId]?.unsubscribe();
```

Comment:
hm, i tried this and the jest tests blew up

Reply:
it seems like I was luckier